webapp/channels/src/components/channel_header/channel_header_title.tsx

```diff
@@ -142,7 +145,9 @@ const ChannelHeaderTitle = ({
                         <button
                             id='channel_header.menuAriaLabel'
                             className={classNames('channel-header__trigger style--none', {active: titleMenuOpen})}
-                            aria-label={intl.formatMessage({id: 'channel_header.menuAriaLabel', defaultMessage: 'Channel Menu'}).toLowerCase()}
+                            aria-label={intl.formatMessage({id: 'channel_header.menuAriaLabel', defaultMessage: '{channelName} Channel Menu'}, {
+                                channelName: isDirect ? channel.display_name + '(you)' : channel.display_name,
+                            }).toLowerCase()}
```

Comment:
This is close, but this has two issues: it includes the `(you)` for any DM channel when that should only be when you DM yourself, and the `(you)` also needs to be translatable as well.

When this is a direct channel with the current user, you'll need to swap `channel_header.menuAriaLabel` for a different string similar to `channel_header.directchannel.you` used in the `ChannelHeaderTitleDirect` component